### PR TITLE
[IPCT1-308] List communities

### PR DESCRIPTION
**Pull request by obernardovieira**
This PR fixes [IPCT1-308] at https://impactmarket.atlassian.net/browse/IPCT1-308

# Description
THis PR creates the base for the new list of communities. It currently does not have filters nor it's linked to any other screen. Should be available form the homepage on mobile app, once it's done. See linked JIRA issues.

### Type of change
- New feature (adds functionality)

# How Has This Been Tested?
- [ ] Manually
  - [ ] [BLU Advance L5](https://www.amazon.com/Advance-A390L-Unlocked-Phone-Camera/dp/B07Z6Q9NCZ/)
  - [ ] [SLIDE SP4514](https://www.amazon.com/dp/B06ZZ4KZF9?psc=1&ref=ppx_yo2_dt_b_product_details)
  - [ ] [Asus ZenFone 3 Max](https://www.tudocelular.com/Asus/fichas-tecnicas/n3260/Asus-ZenFone-3-Max.html)
  - [x] iPhone 6
- [ ] Automated

# Screenshots/Videos
Picture below.

### CODE REVIEW

**Comment by obernardovieira** [comment]
IMG_0006

**Comment by obernardovieira** [comment]
IMG_0019

File: src/views/communities/List.tsx [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+                            <NoSuspiciousSvg />
+                        )}
+                        <Dot />
+                        <Text
+                            style={[
+                                styles.infoText,
+                                { fontFamily: 'Inter-Bold' },
+                            ]}
+                        >
+                            UBI
```
Comment (Jonathansoufer) [inline src/views/communities/List.tsx:113]: Avoid using of hardcoded texts. Use i18n instead ;)

File: src/views/communities/List.tsx [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+    const exchangeRates = useSelector(
+        (state: IRootState) => state.app.exchangeRates
+    );
+    const userCurrency = useSelector(
+        (state: IRootState) => state.user.metadata.currency
+    );
+
+    const [communtiesOffset, setCommuntiesOffset] = useState(0);
+    // TODO: use later with filters
+    const [communtiesOrder, setCommuntiesOrder] = useState('bigger');
```
Comment (Jonathansoufer) [inline src/views/communities/List.tsx:159]: Maybe we should use constants instead of direct texts here, but not critical.

**Review by Jonathansoufer — CHANGES_REQUESTED** [review]

**Comment by obernardovieira** [comment]
@Jonathansoufer ?

**Review by Jonathansoufer — APPROVED** [review]
LGTM! I didn't received a new request to review it, so that's the delay. Approved.

**Pull request merged**